exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js

```diff
@@ -43,14 +44,14 @@ module.exports = function() {
 
 
 
-  this.When(/^receiving the message "([^"]*)"$/, function(message) {
+  When(/^receiving the message "([^"]*)"$/, function(message) {
     this.exocom.send({ service: '_____serviceRole_____',
                        name: message })
   })
 
 
-  this.When(/^receiving the message "([^"]*)" with the payload:$/, function(message, payload, done) {
-    this.fillIn_____modelName@camelcase_____Ids(payload, (filledPayload) => {
+  When(/^receiving the message "([^"]*)" with the payload:$/, function(message, payload, done) {
+    this.fillIn_____modelNameCamelcase_____Ids(payload, (filledPayload) => {
```

Comment:
This function is defined in `support/world.js` and the logic that involves `tmplcov` is there. I only changed the function name to fix the syntax error. The issue is that the feature tests for `exoservice-es6-mongodb` and `exoservice-ls-mongodb` both failed originally in the master version.